src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -491,22 +491,21 @@ class GroupReadsByUmi
     rec.mapq >= minMapQ && mateMqOk
   }
 
+  // A handful of counters for tracking reads
```

Comment:
Reproducing top-level comments from my last review here since you seem to have missed them and I can't comment on the appropriate lines since they are unmodified:

> canTakeNextGroupByUmi on line 480 will, I think, need updating. If the sort was done externally then I don't think we can rely on the fact that they records are also sorted by the same UMI sequence we're going to look at, and so it's not safe to take the next group by UMI.

> Usage needs updating to clearly explain when sorting will and won't happen.